torch/_inductor/config.py

```diff
@@ -56,15 +56,26 @@ def bundled_autotune_remote_cache_default() -> Optional[bool]:
 # enable autotune local cache
 autotune_local_cache = True
 
-# enable autotune remote cache
+# Enable autotune remote cache.
+#
+# Enables/disables the autotune remote cache regardless of the state of
+# autotune_local_cache. If both local and remote are enabled then on write both
+# are written and on read local is checked first and only on a cache miss is
+# remote read.
+#
 # False: Disables the cache
 # True: Enables the cache
 # None: Not set -- Off for OSS, JustKnobs based for internal
 autotune_remote_cache: Optional[bool] = autotune_remote_cache_default()
 
-# enable bundled autotune cache
+# Enable bundled autotune cache.
+#
+# The bundled_autotune_remote_cache depends on the local cache for local state
+# management. As a result if the local cache is disabled this will also disable
+# the bundled autotune cache.
```

Comment:
I still think it's better to explicitly say that this does nothing for remote cache, so I'm not left guessing if you intentionally omitted it or if it was an oversight.